var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -353,6 +367,10 @@ def plugin_cmake_variant(plugin_name, spack_variant):
         if spec.satisfies("@18: +python"):
             args.append("-DACTS_USE_SYSTEM_PYBIND11=ON")
 
+        if spec.satisfies("@20.1: +svg"):
+            args.append("-DACTS_SETUP_ACTSVG=ON")
```

Comment:
Removing `ACTS_SETUP_ACTSVG` (as removed in https://github.com/acts-project/acts/pull/1573)
```suggestion
```